src/test_lib_json/main.cpp

```diff
@@ -1996,6 +1996,30 @@ JSONTEST_FIXTURE(StreamWriterTest, writeZeroes) {
   }
 }
 
+JSONTEST_FIXTURE(StreamWriterTest, unicode) {
+  // Create a Json value containing UTF-8 string with some chars that need escape (tab,newline).
+  Json::Value root;
+  root["test"] = "\t\n\xF0\x91\xA2\xA1\x3D\xC4\xB3\xF0\x9B\x84\x9B\xEF\xBD\xA7";
+
+  Json::StreamWriterBuilder b;
+
+  // Default settings - should be unicode escaped.
+  JSONTEST_ASSERT(Json::writeString(b, root) == "{\n\t\"test\" : \"\\t\\n\\ud806\\udca1=\\u0133\\ud82c\\udd1b\\uff67\"\n}");
+
+  // Enable UTF-8
```

Comment:
Nit: remove comments that just state what the line of code below them does, especifically the enable and disable UTF-8 comments.

Reply:
Done.